src/components/ExampleComponent/ExampleComponent.js

```diff
@@ -4,14 +4,14 @@ import { withFirebase } from "utils/Firebase";
 const ExampleComponent = ({ firebase }) => {
   const getData = () =>
     firebase
-      .getLesson()
+      .customLessons("AxtySwFjYwR0uEsyP3Ds9nO22CY2")
       .get()
-      .then(querySnapshot => {
-        return querySnapshot.docs.map(doc =>
-          Object.assign(doc.data(), { id: doc.id })
-        );
-      });
-
+      .then(snapshot =>
+        snapshot.docs.map(d => {
+          const data = d.data();
+          return { id: d.id, ...data };
+        })
```

Comment:
once the other pr gets merged in this probably wont be a thing or itll cause conflicts just a note